ocaml/stdlib/dune

```diff
@@ -186,6 +186,7 @@
     weak.ml
     weak.mli
 
+    ocaml_compiler_internal_params
```

Comment:
I don't think this should be in the `stdlib` dune file -- it isn't related to the stdlib.

Reply:
That's where the compiler expects it to be installed.